service.libraryautoupdate/addon.xml

```diff
@@ -0,0 +1,88 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.libraryautoupdate"
+    name="Library Auto Update" version="1.0.4" provider-name="robweber">
+  <requires>
+    <import addon="xbmc.python" version="2.25.0"/>
+  </requires>
+  <extension point="xbmc.python.script" library="manual.py"> 
+     <provides>executable</provides> 
+  </extension> 
+  <extension point="xbmc.service" library="default.py" start="login">
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="be">Update your Kodi Video and Music Libraries on a timer. Timer runs as an Kodi service so you never miss an update.</summary>
```

Comment:
All these attributes need to be updated. See here the values: https://github.com/xbmc/xbmc/tree/master/addons/skin.estuary/language

Old default (en) is now en_gb

Reply:
Please also take the time to change XBMC to Kodi